src/main/java/org/jboss/staxmapper/XMLMapperImpl.java

```diff
@@ -100,10 +108,24 @@ public void deparseDocument(XMLContentWriter contentWriter, XMLStreamWriter stre
         contentWriter.writeContent(new FormattingXMLStreamWriter(streamWriter));
     }
 
-    @SuppressWarnings({ "unchecked" })
+    private XMLElementReader getParser(final QName name) {
```

Comment:
Not sure why github still shows old change. It is fixed and signature has <?> in it, but this (old) review still shows original change.